src/driver/driver.cpp

```diff
@@ -75,6 +75,16 @@ DriverStatus EvolutionDriver::Execute() {
   // Defaults must be set across all ranks
   DumpInputParameters();
 
+  // Before loop do work
```

Comment:
How about adding some profiling regions around (just for completeness)?

Reply:
Good point. Done. Sorry @jdolence this means that if my PR goes in before #969 there will be one exposed profiling region in here that we would either need to ignore or change.